import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/backupuploader/GcpUtil.java

```diff
@@ -0,0 +1,58 @@
+package com.google.cloud.healthcare.imaging.dicomadapter.backupuploader;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class GcpUtil {
+    private String path;
+    private String project;
+    private String bucket;
+    private String obj;
+
+    public GcpUtil(String path) {
+        this.path = path;
+    }
+
+    public void parse() throws GcpParseException {
+        try {
+            String route = this.path.replaceAll("gs://", "");
+            String[] ar = route.split("/");
```

Comment:
https://docs.oracle.com/javase/tutorial/essential/io/pathOps.html It`s more new. With help of this API you can create more readable code.